data/src/main/java/com/linkedin/data/schema/PdlBuilder.java

```diff
@@ -258,6 +261,8 @@ private static String escapeIdentifier(String identifier)
    */
   abstract PdlBuilder writeJson(Object value) throws IOException;
```

Comment:
If you agree with my suggestion below, then this method should no longer be abstract and it instead should just delegate to `writeJson(value, null)`